src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import helper.*;
```

Comment:
I think the use of * in import statements are not allowed. Please consider listing down everything instead of using *?